docs/source/_json/registry_get_list.resp

```diff
@@ -12,26 +12,70 @@ Content-Type: application/json
   "items": [
     {
       "name": "Products.CMFPlone.i18nl10n.override_dateformat.Enabled", 
+      "schema": {
```

Comment:
I get what your saying. I've added this to 'schema' because it uses a JsonSchemaProvider and produces a lot more information than just title description and type.
See for example https://github.com/plone/plone.restapi/pull/363/files#diff-1cd823c9eb155c78b8fd8be2b6fdc818R180